### PR TITLE
commands.doctor: add some colors to the output

### PR DESCRIPTION
**Pull request by alexfikl**
Added some colors to the output (that largely match what's in `papis.logging`). It looks like this in my terminal

Screenshot_20230909_110518

@jghauser What do you think? For a lot of errors, the output was hard to parse at a glance, so I thought some colors would do it good!

### CODE REVIEW

**Comment by jghauser** [comment]
Looks great and rally helps understanding what's going on even at a glance. Thanks! :blush:

**Comment by alejandrogallo** [comment]
It makes sense! Ship it! :D

**Comment by alexfikl** [comment]
Thanks a lot for taking a look! :heart:
